generate_events.py

```diff
@@ -1606,19 +1656,259 @@ def define_sex_events(es: EventMap):
+                          You continue riding him, but as you slow down for a moment {THEM} pushes you backwards, trying to get control over you.
+                          """),
+                   Option(EventsSex.REVERSE_COWGIRL, OptionCategory.DOM,
+                          "Roll around",
+                          transition_text=f"""
+                          You turn your back towards {THEM}, treating him as no more than a piece of meat.""",
+                          failed_transition_text=f"""
+                          You begin to turn your back towards him, but your legs feel weak and {THEM} doesn't miss his opportunity to take the lead and guide your next move.
+                          """),
+                   Option(EventsSex.HOTDOG_ON_TOP, OptionCategory.SUB,
+                          "Pull out to recover from his thrusting",
+                          dom_success_adjustment=10,
+                          transition_text=f"""
+                          {THEM} twitches backwards, his rod now between your buns, as he almost escapes from under you.""",),
+                   Option(EventsSex.MISSIONARY, OptionCategory.SUB,
+                          "Lie on your back and let him pound you",
+                          transition_text=f"""
+                          You end up on your back, readying yourself as {THEM} lifts your legs and enters you from above.""",),
+                   Option(EventsCum.PULL_OUT_CUM_ON_ASS, OptionCategory.DOM,
+                          "Make him pull out and cum on your ass",
+                          transition_text=f"""
+                          You put your hand on his rod, guiding him to unload on your cheeks instead""",
+                          failed_transition_text=f"""
+                          You put your hand on his rod, trying to guide him to cum on your cheeks instead, but are suddenly grabbed."""),
+                   Option(EventsCum.CREAMPIE_ON_TOP, OptionCategory.SUB,
+                          "Pin him down and take his seed",
+                          transition_text=f"""
+                          "Your body tightens as he begins to twitch inside you.""",
+                          ),
+                   
+               )))
+    es.add(Sex(EventsSex.MISSIONARY, "Lie on Your Back", #Added by me#
+               stam_cost_1=2.5, stam_cost_2=2,
+               root_become_more_sub_chance=10,
+               animation_left=FLIRTATION_LEFT, animation_right=FLIRTATION_LEFT,
+               root_removes_clothes=True, partner_removes_clothes=True,
+               desc=ComposedDesc(f"""
+               You lie there, legs on his shoulders, each of {THEM}'s vigorous thrusts slapping loudly against your cheeks as
+               waves of pleasure course through your core.\\nYou can hardly control yourself
+               """,
+               TriggeredDesc(f"{HAS_TRAIT} = {LIDA_SUB}", """
+                and you wish you had even #sub less#!"""),
```

Comment:
even less control* 
`control yourself` doesn't actually set "control" as a known since control there is used as a verb.